### DISCUSSION TITLE
Can lightweight charts add new data from stream?

### DISCUSSION THREAD

**Discussion by neobie** [tradingview/lightweight-charts#777] [General]
Can lightweight chart add / update latest bar as  live update base on stream socket / interval update?
Any example provided?

**Comment by neobie** [comment]
Saw jsfiddle.  how to delete this discussion ?

**Reply by timocov** [reply]
You can post the answer so others might find it

**Comment by chung362** [comment]
Official example:  Realtime emulation
https://www.tradingview.com/lightweight-charts/

https://jsfiddle.net/TradingView/yozeu6k1/

**Comment by geo1590** [comment]
Any python examples?
